packages/bootstrap-vue-next/src/components/BFormCheckbox/BFormCheckbox.vue

```diff
@@ -87,9 +87,8 @@ const props = withDefaults(
 )
 
 const emit = defineEmits<{
-  'change': [value: CheckboxValue | CheckboxValue[]]
-  'input': [value: CheckboxValue | CheckboxValue[]]
-  'update:modelValue': [value: CheckboxValue | CheckboxValue[]]
+  change: [event: Event]
+  input: [event: Event]
```

Comment:
```suggestion
  input: [event: Event]
  'update:modelValue': [value: CheckboxValue | CheckboxValue[]]
```
The model value update is still required